packages/cozy-client/src/models/file.js

```diff
@@ -61,3 +63,13 @@ export function ensureFilePath(file, parent) {
   const path = parent.path + '/' + file.name
   return { path: path, ...file }
 }
+
+/**
+ * Get the id of the parent folder
+ *
+ * @param {object} file  - io.cozy.files document
+ * @returns {string|undefined} id of the parent folder, if any
```

Comment:
What does it mean if there is no parent folder ?

Reply:
It is usualy that youare at the root folder (or that your files database is somehow broken)

Reply:
Mmh I thought we had an id for the root folder, was I mistaken @nono ?

Reply:
We have a root (with the id `io.cozy.files.root-dir`), and it has no parent.

Reply:
Oh, I was wrong about the root having an undefined dir_id. It is defined to an empty string.

Let me fix this method so it returns `null` when at root.